include/crocoddyl/core/actions/diff-lqr.hpp

```diff
@@ -39,6 +39,10 @@ class DifferentialActionModelLQRTpl : public DifferentialActionModelAbstractTpl<
   virtual boost::shared_ptr<DifferentialActionDataAbstract> createData();
   virtual bool checkData(const boost::shared_ptr<DifferentialActionDataAbstract>& data);
 
+  virtual void quasiStatic(const boost::shared_ptr<DifferentialActionDataAbstract>& data, Eigen::Ref<VectorXs> u,
```

Comment:
Why do you implement it as empty function?
If you do not plan to do it, then remove this function declaration so it will run the default `quasiStatic` method.